popmon/visualization/templates/core.html

```diff
@@ -16,6 +16,7 @@
     {% include 'assets/css/bootstrap.min.css' %}
     {% include 'assets/css/custom-style.css' %}
   </style>
+  
```

Comment:
Newline can be removed now that javascript is moved to the footer (then the file won't show up in the diff)